modules/dbnd/src/dbnd/_core/task_run/task_run_sync_local.py

```diff
@@ -50,11 +55,14 @@ def __init__(self, task_run):
 
     def _local_cache_target(self, target_):
```

Comment:
When calling `_local_cache_target` you are in compile time (from `__init__`).
This means that the driver is the currently executing task.
This means that your "local" environment -- is the driver's local environment, NOT THE TASK'S ENV!

To be blunt -- we need to create the relevant path (and call `os.makedirs` most importantly) only in execution time, not in compile time (to ensure local env is really the task's local env, for example when executing on kubernetes -- driver runs in pod A, creates dir `/tmp/A`, transfers run to pod B, then pod B doesn't have that path! ERROR!

Reply:
I hope I fixed it